scripts/init-formulas.sh

```diff
@@ -105,7 +105,7 @@ clone_update() {
     # tell the minion where it can find the formula's state and pillar data
     echo "    - $repo_path/salt/" >> /etc/salt/minion.d/file_roots.conf
 
-    topfile="$repo_path/salt/example.top"
+    topfile="$repo_path/salt/${BUILDER_TOPFILE-example.top}"
```

Comment:
I hadn't seen this syntax before.

Reply:
it's the best way I found to specify defaults for a variable